src/Views/ErrorView.vala

```diff
@@ -61,6 +61,8 @@ public class ErrorView : AbstractInstallerView {
 
         var grid = new Gtk.Grid ();
         grid.row_spacing = 6;
+        grid.margin_left = 24;
+        grid.valign = Gtk.Align.CENTER;
```

Comment:
`margin_left` is deprecated and not RTL friendly; use `margin_start` instead. Also, I think these are pop-specific styling?